crimsobot/cogs/cringo.py

```diff
@@ -186,24 +190,24 @@ async def cringo_main(self, ctx: CrimsoContext, card_size: int = 4) -> None:
         if ctx.guild and ctx.guild.id == 552650672965943296:
             nerf = (2 - SERVER_BONUS) * nerf
 
-        game_is_cursed = name_prefix == 'None'
-        score_string, winner = await cringo.cringo_scoreboard(list_of_players, nerf, game_is_cursed)
+        sorted_players = sorted(list_of_players, key=lambda item: item.score, reverse=True)
+        score_string = await cringo.cringo_scoreboard(sorted_players, game_is_cursed, True)
+
+        for index, player in enumerate(list_of_players):
+            # calculate and give out winnings, but give nothing to the cursed
+            player.winnings = 0 if game_is_cursed else round(player.score / nerf, 1)
```

Comment:
so previously we were depositing the full value, now it's just to one decimal point.
i'm fine with this but wanted to call it out.